content/sensu-core/2.0/reference/silencing.md

```diff
@@ -0,0 +1,196 @@
+example      | {{< highlight shell >}}"check": "haproxy_status",
+"subscription" : "load_balancer"{{</highlight>}}
+
+subscription | 
+-------------|------ 
+description  | The name of the subscription the entry should match 
+required     | true, unless `check` is provided
+type         | String
+example      | {{< highlight shell >}}"subscription": "agent:i-424242"{{</highlight>}}
+example      | {{< highlight shell >}}
+  "subscription": "agent:i-424242",
+  "check" : "haproxy_status"
+{{</highlight>}}
+
+id           | 
+-------------|------ 
+description  | Silencing identifier generated from the combination of a subsription name and check name. 
+required     | false - this value cannot be modified 
+type         | String
+example      | {{< highlight shell >}}
+  "id": "appserver:mysql_status",
+  "check": "mysql_status",
+  "subscription": "appserver",
+{{</highlight>}}
+
+begin        | 
+-------------|------ 
+description  | Time at which silence entry goes into effect, in epoch. 
+required     | false 
+type         | Integer 
+example      | {{< highlight shell >}}
+  "begin": 1512512023,
+  "check": "disk_utilization",
+  "subscription": "agent:i-424242"
+{{</highlight>}}
+
+expire       | 
+-------------|------ 
+description  | Number of seconds until this entry should be deleted. 
+required     | false 
+type         | Integer 
+default      | -1
+example      | {{< highlight shell >}}
+  "expire": 3600,
+  "check": "disk_utilization",
+  "subscription": "agent:i-424242"
+{{</highlight>}}
+
+expire_on_resolve       | 
+-------------|------ 
+description  | If the entry should be deleted when a check begins return OK status (resolves). 
+required     | false 
+type         | Boolean 
+default      | false 
+example      | {{< highlight shell >}}
+  "expire_on_resolve": true,
+  "check": "disk_utilization",
+{{</highlight>}}
+
+
+creator      | 
+-------------|------ 
+description  | Person/application/entity responsible for creating the entry. 
+required     | false 
+type         | String 
+default      | null 
+example      | {{< highlight shell >}}
+  "creator": "Application Deploy Tool 5.0",
+  "subsription": "appservers",
+  "check": "app_status"
+{{</highlight>}}
+
+reason       | 
+-------------|------ 
+description  | Explanation for the creation of this entry.
+required     | false 
+type         | String 
+default      | null 
+example      | {{< highlight shell >}}
+  "creator": "mercedes",
+  "subsription": "webserver",
+  "check": "app_status",
+  "reason": "rebooting the world"
+{{</highlight>}}
+
+## Examples
+
+### Silence all checks on a specific entity 
+Assume a Sensu agent `i-424242` which we wish to silence any alerts on. We’ll do this by taking advantage of per-agent subscriptions:
+{{< highlight json >}}
+{
+  "expire": -1,
+  "expire_on_resolve": false,
+  "creator": null,
+  "reason": null,
+  "check": null,
+  "subscription": "agent:i-424242",
+  "id": "agent:i-424242:*"
+}
+{{ </highlight> }}
```

Comment:
Looks like these spaces are causing a weird problem in my local installation of Hugo and the page won't stop loading. It probably need to be fixed for all examples.

![screen shot 2018-03-14 at 10 08 14 am](https://user-images.githubusercontent.com/4105580/37407325-c0424bd6-276f-11e8-922b-56f5193ceb45.png)